test/test_genbank_sbol3_direct.py

```diff
@@ -133,6 +133,18 @@ def test_sbol3togb_2(self):
         )
         self._test_sbol3_to_genbank(sample_sbol3_file=sbol3_file, sample_genbank_file=genbank_file)
 
+    def test_round_trip_testfile_1(self):
```

Comment:
I'd like to see these two tests become a single test that iterates over all files with a `.gb` extension.

Typing from memory, so this may be a bit off, but should give you a feel for what I'm suggesting:

```python
test_file_dir = Path(__file__).parent / 'test_files'
for genbank_file in test_file_dir.glob('*.gb'):
    self._test_round_trip_genbank(genbank_file)
```

You will want to verify that it is in fact doing what I think it does, so you could print the file names temporarily to make sure.

This has the advantage that as genbank files are added to the `test_files` directory they will automatically be tested for successful round tripping.

Reply:
Okay right, fixed this